hw/formal/tools/dvsim/common_formal_cfg.hjson

```diff
@@ -22,14 +22,15 @@
 
   // TODO: switch the tool to formal once the corresponding edalize backend is avaiable
 
+  target:            formal
```

Comment:
'Target' is a super-overloaded term used by different tools to mean different things (for dvsim, target means build, run etc., for fusesoc, its the core, for SW its the application ...). Perhaps call this `fusesoc_target` to explicitly tell what it refers to.

Reply:
Good point, thanks Sri, let me update that